tools/sync_test_server/ros-testing-server.js

```diff
@@ -40,50 +41,65 @@ function startRealmObjectServer(done) {
     // https://github.com/realm/realm-object-server/issues/1297
     var logFindingCounter = 2
 
-    stopRealmObjectServer();
-    temp.mkdir('ros', function(err, path) {
-        if (!err) {
-            winston.info("Starting sync server in ", path);
-            var env = Object.create( process.env );
-            winston.info(env.NODE_ENV);
-            env.NODE_ENV = 'development';
-            syncServerChildProcess = spawn('realm-object-server',
-                    ['--root', path,
-                    '--configuration', '/configuration.yml'],
-                    { env: env});
-            // local config:
-            syncServerChildProcess.stdout.on('data', (data) => {
-                if (logFindingCounter != 0 && /client: Closing Realm file: .*__auth.realm/.test(data)) {
-                    if (logFindingCounter == 1) {
-                        done()
+    stopRealmObjectServer(function(err) {
+        if(err) {
+          return;
+        }
+        temp.mkdir('ros', function(err, path) {
+            if (!err) {
+                var oldCwd = process.cwd();
+                process.chdir(path);
```

Comment:
spawn can set the `cwd` as well, would it be better to pass the `path` there instead of change the current process's `cwd`? https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options